core/src/test/java/com/datastax/oss/driver/api/core/cql/StatementBuilderTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.api.core.cql;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class StatementBuilderTest {
+
+  private static class NullStatementBuilder
+      extends StatementBuilder<NullStatementBuilder, SimpleStatement> {
+
+    public NullStatementBuilder() {
+      super();
+    }
+
+    public NullStatementBuilder(SimpleStatement template) {
+      super(template);
+    }
+
+    @Override
+    public SimpleStatement build() {
+      return null;
+    }
+  }
+
+  @Test
+  public void should_handle_set_tracing_without_args() {
+
+    NullStatementBuilder builder = new NullStatementBuilder();
+    assertFalse(builder.tracing);
```

Comment:
Nit: we prefer AssertJ for assertions.
```
import static com.datastax.oss.driver.Assertions.assertThat;
...
assertThat(builder.tracing).isFalse()
```
Not a huge difference for a simple boolean, but for more complex types like collections it's way more expressive.

Reply:
This makes sense.  I saw the use of AssertJ elsewhere but I went to the JUnit ones out of muscle memory.  There's no reason to make a special case for this one test, though, so I'll update to use AssertJ.